clientv3/client.go

```diff
@@ -93,7 +93,10 @@ func NewFromURL(url string) (*Client, error) {
 // Close shuts down the client's etcd connections.
 func (c *Client) Close() error {
 	c.cancel()
-	c.Watcher.Close()
+	err := c.Watcher.Close()
+	if err != nil {
```

Comment:
`Close` should probably keep going regardless of whether if there's an error, otherwise it'll leak lease resources if there's any watch error